clusterloader2/pkg/measurement/common/probes/manifests/metricsServer/metrics-server-prober-deployment.yaml

```diff
@@ -4,27 +4,27 @@ metadata:
   namespace: probes
   name: metrics-server-prober
   labels:
-    probe: metrics-server-probe
+    probe: kube-client
 spec:
   selector:
     matchLabels:
-      probe: metrics-server-probe
+      probe: kube-client
   replicas: {{.Replicas}}
   template:
     metadata:
       labels:
-        probe: metrics-server-probe
+        probe: kube-client
     spec:
       containers:
         - name: prober
-          image: gcr.io/k8s-testimages/probes:v0.0.4
+          image: gcr.io/k8s-testimages/probes:v0.0.5
```

Comment:
@mm4tt, I build the image using `https://github.com/kubernetes/perf-tests/blob/master/util-images/probes/Dockerfile ` but don't have permission to push `gcr.io/k8s-testimages/probes:v0.0.5`.
Need help to push an image `gcr.io/k8s-testimages/probes:v0.0.5`

Reply:
/cc @wojtek-t @mborsz @mm4tt